test/3x/torch/quantization/weight_only/test_rtn.py

```diff
@@ -1,136 +1,166 @@
-import unittest
+import copy
 
+import pytest
```

Comment:
Our current UTs are based on [unittest](https://docs.python.org/3/library/unittest.html). @chensuyue, is it acceptable to include two UT frameworks in a one project? If so, please remember to include `pytest` in both requirements.txt and the BoM.

Reply:
ok, suyue aware this.